server/lib/flow-event.js

```diff
@@ -56,12 +57,20 @@ function isDNT (request) {
   return request.headers.dnt === '1';
 }
 
-function limitLength (param) {
-  if (param && param.length > MAX_PARAM_LENGTH) {
-    return param.substr(0, MAX_PARAM_LENGTH);
+function limitLength (data) {
+  if (data && data.length > MAX_DATA_LENGTH) {
+    return data.substr(0, MAX_DATA_LENGTH);
+  }
+
+  return data;
+}
+
+function sanitiseData (data) {
+  if (data === 'none') {
```

Comment:
yea it comes from https://github.com/mozilla/fxa-content-server/blob/b58f42740541ed6ea3cd41f699ab7eb4231a6cf3/app/scripts/lib/metrics.js#L65 probably